src/cmd/request.go

```diff
@@ -33,9 +33,8 @@ func init() {
 
 var makeRequest = &cobra.Command{
```

Comment:
@Ulexus I noticed that running `turnkey request` with an empty body will hang. Is this expected behavior? If not, I wonder if we should return an error if an empty body is detected

Reply:
It is probably defaulting to looking for input from stdin.  I think that's a pretty decent default behaviour, but it should also be documented, if so, so that it is not unexpected.

Reply:
Yep, the default is stdin.  `--help` shows that